Core/include/Acts/Geometry/LayerCreator.hpp

```diff
@@ -65,6 +65,8 @@ class LayerCreator {
                         std::unique_ptr<const Logger> logger =
                             getDefaultLogger("LayerCreator", Logging::INFO));
 
+  LayerCreator(LayerCreator&&) = default;
+
   /// Destructor
   ~LayerCreator() = default;
```

Comment:
alternatively you can just drop the explicit default destructor. then the compiler will create move constructor and move assignment by itself. right now you would be missing the move assignment

Reply:
You mean dropping the `explicit`?

Reply:
```suggestion
```

I think this way you should get implicit move constructor and assignment

Reply:
A I misread de/constructor,  that makes sense